core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -530,6 +531,18 @@ private static String formatFragment(
                                 formatDouble(outputBufferUtilization.get().getP99() * 100),
                                 formatDouble(outputBufferUtilization.get().getMax() * 100)));
             }
+
+            if (verbose) {
+                TDigest taskOutputDistribution = new TDigest();
+                stageInfo.get().getTasks().forEach(task -> taskOutputDistribution.add(task.getStats().getOutputPositions()));
+                TDigest taskInputDistribution = new TDigest();
+                stageInfo.get().getTasks().forEach(task -> taskInputDistribution.add(task.getStats().getProcessedInputPositions()));
```

Comment:
Any particular reason for preferring `processedInputPositions` over `processedInputDataSize` ?